### PR TITLE
Adding new base configuration files

### PR DESCRIPTION
**Pull request by pardhu-pokuri**
Adding base configuration files for tomcat web settings, java security override and catalina properties

### CODE REVIEW

File: charts/pega/config/deploy/tomcat-web.xml
Comment (Saurabh-16) [inline charts/pega/config/deploy/tomcat-web.xml:1]: Can we make this file candidate for xml lint check. Sometimes we unintentionally disturb the xml formatting resulting in xml parsing issue.
See reference here
https://github.com/pegasystems/pega-helm-charts/blob/master/validatexml.sh

File: charts/pega/config/deploy/catalina.properties [hunk 1]
```diff
@@ -0,0 +1,219 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (Saurabh-16) [inline charts/pega/config/deploy/catalina.properties:1]: I am assuming we have taken the source code from tomcat container we run during deployments?

Comment (saikarthik528) [reply charts/pega/config/deploy/catalina.properties:1]: Correct.  We picked the default file

File: charts/pega/config/deploy/tomcat-web.xml
Comment (saikarthik528) [reply charts/pega/config/deploy/tomcat-web.xml:1]: Addressed

**Review by Saurabh-16 — APPROVED** [review]

**Review by kishorv10 — APPROVED** [review]

**Pull request merged**